common/djangoapps/third_party_auth/migrations/0004_auto_20200919_0955.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.16 on 2020-09-19 09:55
```

Comment:
@gabor-boros I think you added this migration by mistake?

Reply:
@toxinu this addition is intentional. The help_text attribute of the field was changed, hence Django detected that a new migrations needed

Reply:
Oh right 👍🏻